soda/core/tests/data_source/test_automated_monitoring.py

```diff
@@ -17,9 +15,8 @@ def test_automated_monitoring(scanner: Scanner):
               tables:
                 - include SODATEST_%
                 - exclude PROD%
-              find anomalies:
-                row count: true
-                schema: true
         """
     )
+    scan._is_experimental_auto_monitoring = False
```

Comment:
did you mean to leave this to `False`? I would assume this would skip the test. Unless that is what you want?

Reply:
And why not use `@pytest.mark.skip(reason="feature not implemented")` instead?

Reply:
Ah wait sorry, you want to run the test but make sure that it successfully doesn't run the automated monitoring right?